migtests/tests/pg/assessment-report-test-uqc/expectedAssessmentReport.json

```diff
@@ -235,56 +238,56 @@
 		{
 			"ConstructTypeName": "Aggregate Functions",
 			"Query": "SELECT range_intersect_agg(event_range) AS intersection_of_ranges\nFROM sales.events",
-			"DocsLink": "",
+            "DocsLink": "https://docs.yugabyte.com/preview/yugabyte-voyager/known-issues/postgresql/#postgresql-12-and-later-features",
 			"MinimumVersionsFixedIn": null
 		},
 		{
 			"ConstructTypeName": "Aggregate Functions",
 			"Query": "SELECT range_agg(event_range) AS union_of_ranges\nFROM sales.events",
-			"DocsLink": "",
+			"DocsLink": "https://docs.yugabyte.com/preview/yugabyte-voyager/known-issues/postgresql/#postgresql-12-and-later-features",
 			"MinimumVersionsFixedIn": null
 		},
 		{
 			"ConstructTypeName": "Aggregate Functions",
 			"Query": "SELECT\n        any_value(name) AS any_employee\n    FROM employees",
-			"DocsLink": "",
+			"DocsLink": "https://docs.yugabyte.com/preview/yugabyte-voyager/known-issues/postgresql/#postgresql-12-and-later-features",
 			"MinimumVersionsFixedIn": null
 		},
 		{
 			"ConstructTypeName": "Merge Statement",
 			"Query": "MERGE INTO sales.customer_account ca\nUSING sales.recent_transactions t      \nON t.customer_id = ca.customer_id\nWHEN MATCHED THEN\n  UPDATE SET balance = balance + transaction_value\nWHEN NOT MATCHED THEN\n  INSERT (customer_id, balance)\n  VALUES (t.customer_id, t.transaction_value)",
-			"DocsLink": "",
+            "DocsLink": "https://docs.yugabyte.com/preview/yugabyte-voyager/known-issues/postgresql/#merge-command",
 			"MinimumVersionsFixedIn": null
 		},
 		{
 			
 			"ConstructTypeName": "Jsonb Subscripting",
 			"Query": "SELECT \n    data,\n    data[$1] AS name, \n    (data[$2]) as active\nFROM sales.test_json_chk",
-			"DocsLink": "",
+            "DocsLink": "https://docs.yugabyte.com/preview/yugabyte-voyager/known-issues/postgresql/#jsonb-subscripting",
 			"MinimumVersionsFixedIn": null
 		},
 		{
 			"ConstructTypeName": "Jsonb Subscripting",
 			"Query": "SELECT (sales.get_user_info($1))[$2] AS user_info",
-			"DocsLink": "",
+            "DocsLink": "https://docs.yugabyte.com/preview/yugabyte-voyager/known-issues/postgresql/#jsonb-subscripting",
 			"MinimumVersionsFixedIn": null
 		},
 		{
 			"ConstructTypeName": "Jsonb Subscripting",
 			"Query": "SELECT (jsonb_build_object($1, $2, $3, $4, $5, $6) || $7)[$8] AS json_obj",
-			"DocsLink": "",
+            "DocsLink": "https://docs.yugabyte.com/preview/yugabyte-voyager/known-issues/postgresql/#jsonb-subscripting",
 			"MinimumVersionsFixedIn": null
 		},
 		{
 			"ConstructTypeName": "Jsonb Subscripting",
 			"Query": "SELECT ($1 :: jsonb)[$2][$3] as b",
-			"DocsLink": "",
+            "DocsLink": "https://docs.yugabyte.com/preview/yugabyte-voyager/known-issues/postgresql/#jsonb-subscripting",
 			"MinimumVersionsFixedIn": null
 		},
 		{
 			"ConstructTypeName": "Json Type Predicate",
 			"Query": "SELECT * \nFROM sales.json_data\nWHERE array_column IS JSON ARRAY",
-			"DocsLink": "",
```

Comment:
Maybe not right now, but it's worth thinking about whether we actually need to validate DocsLink or not. It's not statically defined for each issues, so it doesn't really add a lot of value to validate this.

Reply:
yeah, probably we can ignore the validation of these. cc: @shubham-yb